File: app/src/test/java/com/worldbuilder/mapgame/MapUtilsTest.java
package com.worldbuilder.mapgame;

import junit.framework.TestCase;

import java.util.ArrayList;

public class MapUtilsTest extends TestCase {

    public void testCreateIndices() {
        ArrayList<Integer> result = MapUtils.createIndices(0, 10);
        assertTrue(result.contains(0));
        assertTrue(result.contains(1));
        assertTrue(result.contains(2));
        assertTrue(result.contains(9));
        assertFalse(result.contains(10));
    }
}

File: app/src/test/java/com/worldbuilder/mapgame/MapUtilsTest.kt
package com.worldbuilder.mapgame

import junit.framework.TestCase
import org.junit.Test

internal class MapUtilsTest {

    @Test
    fun testCreateIndices() {
        val result = MapUtils.createIndices(0, 10)
        TestCase.assertTrue(result.contains(0))
        TestCase.assertTrue(result.contains(1))
        TestCase.assertTrue(result.contains(2))
        TestCase.assertTrue(result.contains(9))
        TestCase.assertFalse(result.contains(10))
    }
}